### ISSUE TITLE
HTML-only changes do not trigger rebuild

### ISSUE THREAD

**Issue by pmi-spease** [djc/askama#210]
Cargo.toml:
```
askama = { version = "0.7.2", features = ["with-actix-web", "serde-json"] }
```

Looks like in the current version of askama you need to add a build.rs:
```
fn main() {
    askama::rerun_if_templates_changed();
}
```

Figured this out while creating the ticket, but I'll leave it here for the time being for anyone else so it's searchable (Feel free to close)

**Comment by botika** [comment]
Fix in #199. In the next version.

**Comment by djc** [comment]
0.8.0 has been released, closing.
